symfit/core/objectives.py

```diff
@@ -329,6 +329,39 @@ def eval_hessian(self, ordered_parameters=[], **parameters):
         return np.atleast_2d(np.squeeze(np.array(result)))
 
 
+class HessianObjectiveJacApprox(HessianObjective):
+    """
+    This object should only be used as a Mixin for covariance matrix estimation.
```

Comment:
Would it not be better to make this a part of `JacobianObjective`? That way there always is a sane way of evalling a Hessian. It would also obliviate the need for `HessApproximation` above.

Reply:
I personally think it is weird for a `JacobianObjective` to implement a hessian. I also think it can lead to false positives elsewhere because we give the minimizer a Hessian if the model has `hessian_model` and the objective has `eval_hessian`. That could trigger a false positive here.